### ISSUE TITLE
Click events don't go through when the camera is during rotation/graph not settled down

### ISSUE THREAD

**Issue by Kert** [vasturiano/3d-force-graph#313]
**Describe the bug**

**To Reproduce**
1. Open the example https://vasturiano.github.io/3d-force-graph/example/click-to-focus/
2. Rotate camera and try pressing on a node in the process (or when the rotation seems to be finished)
3. OR drag any node somewhere and try clicking on any node before the graph changes stop

**Expected behavior**
Clicks events should fire normally (includes onBackgroundClick too)

**Desktop (please complete the following information):**
 - OS: Windows
 - Chrome

**Comment by Alexithemia** [comment]
This happens with the default controls, not sure of a way to optimize, but it doesn't happen with orbit controls (and the controls are simpler in my opinion, as well as working with the camera.)

```
 graph = ForceGraph3D({ controlType: 'orbit' })(containerNode)
```

**Comment by vasturiano** [comment]
@kert are you per chance using a three-finger drag trackpad movement to drag the camera? 

This is a common reason why for a short period after the drag release, clicks are not available. Some trackpads add a 300ms delay after drag release before triggering `pointerup` (which signals the end of the drag interaction), mainly to allow the user to continue the drag motion when briefly lifting the fingers from the trackpad.

You can test that mechanism in this codepen: https://codepen.io/vasturiano/full/PoZebpR

You can also try doing the same using a regular mouse, or a down-press dragging motion on your trackpad, to confirm the root cause of the issue.

**Comment by RaulPROP** [comment]
It also happens with a mouse. As @Alexithemia says, it happens with the default controls (trackball). At least with an older version, I haven't tested the one with the updated controllers yet.

**Comment by vasturiano** [comment]
Ok, I believe this glitch is finally fixed in `v1.66.3`. There was an issue detecting the end of the drag interaction while the controls were still moving the camera via the dampening effect. That's why it was only present with `trackball` controls and not `orbit` as those are not configured to dampen.

This example should now be functioning correctly:
https://vasturiano.github.io/3d-force-graph/example/click-to-focus/

Thanks for the patience and for testing this issue. Got me to fix it at last. 👍
